private/solr-conf/synonyms_und.txt

```diff
@@ -65,20 +38,3 @@ phb,housing bureau
 pbot,transportation
 pdc,prosper portland,portland development commission
 pwb,water
-
-
-# Common spelling corrections
-
-9-1-1=>911
-adress=>address
-artstax=>arts tax
-bball=>basketball
-buisness=>business
-lisence,licens=>license
-complaince,complance=>compliance
-forst=>forest
-paking,oarking=>parking
-tenis=>tennis
-volly ball,vollyball=>volleyball
-volnteer=>volunteer
-witches castle,which's castle=>witch's castle
```

Comment:
I'd recommend keeping this one as we know it's a common misspelling.

Reply:
After reading through the spelling versus synonym documentation, I think we should add some misspelling optimization rather than putting misspellings into our synonyms. I could be convinced otherwise, but I also feel like we have a lot to learn about how to get that working correctly.